### PR TITLE
feat(api): Add bus details to student and teacher creation #171

### PR DESCRIPTION
**Pull request by kural214**

### CODE REVIEW

File: server/database/scripts/api_student_creation.sql [hunk 1]
```diff
@@ -82,7 +85,13 @@ BEGIN
 	 -- Insert the class record
     IF class_id IS NOT NULL OR class_id != 'undefined' THEN
     INSERT INTO StudentClassAllocation (school_id, student_id, class_id, created_by, created_date, active) 
-	 VALUES (school_id, @last_insert_id, class_id, 1, CURRENT_DATE(), active);
+	 VALUES (school_id, @last_insert_id, class_id, created_by, CURRENT_DATE(), active);
+    END IF;
+    
+    -- Insert the bus record
+    IF bus_id IS NOT NULL OR bus_id != 'undefined' THEN
```
Comment (ArunprasadRajendran) [inline server/database/scripts/api_student_creation.sql:92]: Should be AND condition

File: server/database/scripts/api_student_creation.sql [hunk 2]
```diff
@@ -71,7 +73,8 @@ BEGIN
     SET country = JSON_UNQUOTE(JSON_EXTRACT(students_list, CONCAT('$[', i, '].country')));
     SET active = JSON_EXTRACT(students_list, CONCAT('$[', i, '].active'));
  	 SET class_id = JSON_EXTRACT(students_list, CONCAT('$[', i, '].class_id'));
- 	 
+ 	 SET bus_id = JSON_EXTRACT(students_list, CONCAT('$[', i, '].bus_id'));
+ 	 SET created_by = JSON_EXTRACT(students_list, CONCAT('$[', i, '].school_id'));
```
Comment (ArunprasadRajendran) [inline server/database/scripts/api_student_creation.sql:77]: why school_id is set to created_by ?

**Review by ArunprasadRajendran — CHANGES_REQUESTED** [review]

File: server/database/scripts/api_student_creation.sql (hunk 2 above)
Comment (kural214) [reply server/database/scripts/api_student_creation.sql:77]: This change is already there. I am not did this change.

File: server/database/scripts/api_student_creation.sql (hunk 1 above)
Comment (kural214) [reply server/database/scripts/api_student_creation.sql:92]: Modified

**Review by ArunprasadRajendran — APPROVED** [review]

**Pull request merged**